e2e/smoke_test.go

```diff
@@ -2,11 +2,13 @@ package e2e
 
 import (
 	"testing"
+	"time"
 
 	"github.com/coreos/ktestutil/testworkload"
 )
 
 func TestSmoke(t *testing.T) {
+	testworkload.PollTimeoutForNginx = 5 * time.Minute
```

Comment:
I think this can be dropped with the ktestutil change

Reply:
Removing